### PR TITLE
RNAlien

### PR DESCRIPTION
**Pull request by eggzilla**

### CODE REVIEW

File: tools/rna_tools/rnalien/rnalien.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <data format="cm" name="cm" from_work_dir="./outdir/result.cm" label="RNAlien result covariance model"/>
+    <data format="stockholm" name="stockholm" from_work_dir="./outdir/result.stockholm" label="RNAlien result stockholm alignment"/>
+    <data format="fasta" name="sequences" from_work_dir="./outdir/result.fa" label="RNAlien result stockholm alignment"/>
+  </outputs>
+  <tests>
+    <test>
+      <param name="input_sequence" value="U1snRNA.fa"/>
+      <param name="taxonomy_id" value="92525"/>
+      <param name="setup_check" value="True"/>
+      <output name="setupCheck" file="./outdir/setupCheck"/>
```
Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: das hier muss der Pfad relativ zu `test-data` sein ...

File: tools/rna_tools/rnalien/rnalien.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+For further questions please refer to the RNAlien Help page.
+
+.. _Userguide: http://nibiru.tbi.univie.ac.at/rnalien/help _.
+
+    ]]>
+
+  </help>
+  <citations>
+    <citation type="bibtex">@article{doi:10.1093/nar/gkw558,
+author = {Eggenhofer, Florian and Hofacker, Ivo L. and Höner zu Siederdissen, Christian},
```
Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: nehm doch einfach den doi

File: tools/rna_tools/rnalien/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: rnalien
+owner: rnateam
+description: RNAlien unsupervized RNA family model construction
+homepage_url: http://rna.tbi.univie.ac.at/AREsite
+long_description: |
+Determining  the  function  of  a  non-coding RNA
+requires  costly  and  time-consuming  wet-lab  experiments.
```
Comment (bgruening) [inline tools/rna_tools/rnalien/.shed.yml]: Ich glaube das muss eingerückt werden

File: tools/rna_tools/rnalien/rnalien.xml [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                -t $taxonomy_id
+      #end if 
+      -o `pwd`
+      -d outdir
+      -c "\${GALAXY_SLOTS:-4}"
+      -z $z
+      -e $e
+      -b $b
+      $a
+      #if $tax_restriction
```
Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: Mh, tax_restriction is not defined as param.
But, the defined `-s` and `-f` are not defined here.

File: tools/rna_tools/rnalien/rnalien.xml [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+           -r $tax_restriction
+      #end if         
+      #if $setup_check
```
Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: Is also not defined in the arguments. And I would not include it at all as an option.

Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: A Galaxy tool should assume that everything is installed I guess.

File: tools/rna_tools/rnalien/rnalien.xml (hunk 1 above)
Comment (eggzilla) [inline tools/rna_tools/rnalien/rnalien.xml]: Thanks, fixed

File: tools/rna_tools/rnalien/.shed.yml (hunk 3 above)
Comment (eggzilla) [inline tools/rna_tools/rnalien/.shed.yml]: ok, tab indented it

File: tools/rna_tools/rnalien/rnalien.xml (hunk 2 above)
Comment (eggzilla) [inline tools/rna_tools/rnalien/rnalien.xml]: is now just gwk558

Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: eher sowas hier:
```
      <citations>
          <citation type="doi">10.1093/nar/gkw558</citation>
      </citations> 
```

File: tools/rna_tools/rnalien/rnalien.xml [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+      #end if
+      > 'stdoutlog'          
+    ]]>
+  </command>
+  <inputs>
+    <param name="input_sequence" type="data" format="fasta" label="Input sequence"/>
+    <param name="taxonomy_id" type="integer" value="" label="NCBI taxonomy id of input organism" optional="true" help="-t"/>
+    <param argument="-z" type="float" value="1.0" min="0.1" label="Only candidate sequences with a nSCI higher than this value are accepted." help=""/>
+    <param argument="-e" type="float" value="0.001" min="0.000000000001" label="Evalue cutoff for cmsearch filtering" help=""/>
+    <param name="blast_database" type="text" label="Specify name of blast database to use." value="nt" help="Blast databases, e.g. nt, refseq_genomic,.." />
```
Comment (bgruening) [inline tools/rna_tools/rnalien/rnalien.xml]: This should be a select box isn't it? Is this using the online NCBI server?

**Pull request merged**